main.py

```diff
@@ -1,10 +1,46 @@
+from django.db import transaction
+
 import init_django_orm  # noqa: F401
 
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
```

Comment:
Consider adding error handling for JSON loading to manage potential issues like file not found or JSON decoding errors. This will make your code more robust.